frontend/layouts/static-page/header/component.tsx

```diff
@@ -60,87 +71,191 @@ export const Header: React.FC<HeaderProps> = ({
       })}
     >
       <LayoutContainer>
-        <div className="flex items-center justify-between pt-3 pb-3 md:pt-6 md:pb-4 lg:justify-start lg:space-x-10">
+        <div className="flex items-center justify-between pt-3 pb-3 md:pt-6 md:pb-4 lg:space-x-10">
           <Link href="/">
-            <a className="font-semibold">HeCo Invest</a>
+            <a className="font-semibold ">HeCo Invest</a>
           </Link>
-          <div className="flex items-center space-x-2 lg:hidden">
-            <div className="shrink-0">
+          <div className="flex">
+            {/* NAVIGATION MENU LG */}
+            <div className="hidden lg:flex-1 lg:flex lg:items-center lg:justify-end lg:mr-4">
+              <nav className="flex space-x-8">
+                <ActiveLink href="/discover" activeClassName="font-semibold">
+                  <a title={formatMessage({ defaultMessage: 'Search', id: 'xmcVZ0' })}>
+                    <Icon icon={SearchIcon} />
+                  </a>
+                </ActiveLink>
+                <ActiveLink href="/investors" activeClassName="font-semibold">
+                  <a>
+                    <FormattedMessage defaultMessage="For investors" id="MfCYKW" />
+                  </a>
+                </ActiveLink>
+                <ActiveLink href="/project-developers" activeClassName="font-semibold">
+                  <a>
+                    <FormattedMessage defaultMessage="For project developers" id="F1+h/t" />
+                  </a>
+                </ActiveLink>
+                <ActiveLink href="/about" activeClassName="font-semibold">
+                  <a>
+                    <FormattedMessage defaultMessage="About" id="g5pX+a" />
+                  </a>
+                </ActiveLink>
+              </nav>
+            </div>
+
+            {/* SELECT LANGUAGE MENU */}
+            <div className="self-center">
               <LanguageSelector />
             </div>
-            <Menu
-              className="shrink-0"
-              Trigger={
+
+            {/* USER MENU (MD/LG)*/}
+            <div className="items-center hidden sm:flex">
+              {user ? (
+                <Menu
+                  Trigger={
+                    <Button
+                      className="px-2 sm:px-4 focus-visible:outline-green-dark"
+                      theme="naked"
+                      size="small"
+                      aria-expanded={userMenuOpen}
+                      onClick={() => setUserMenuOpen(true)}
+                    >
+                      <div
+                        className={cx('w-8 h-8 rounded-full flex justify-center items-center', {
+                          'bg-green-dark text-white': showBackground,
+                          'text-green-dark bg-white': !showBackground,
+                        })}
+                      >
```

Comment:
It's because the button is composed by the image (or the initials) and the chevronDown, with a transparent background behind them, so I can't apply a theme
https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=2%3A3646

Reply:
@barbara-chaves You're right, ignore my second comment. I didn't realise it was only for the circle with the initials.